pkg/openstack/loadbalancer_test.go

```diff
@@ -534,3 +536,104 @@ func Test_getListenerProtocol(t *testing.T) {
 		})
 	}
 }
+
+func TestLbaasV2_getMemberSubnetID(t *testing.T) {
+	lbaas := &LbaasV2{
+		LoadBalancer: LoadBalancer{
+			opts: LoadBalancerOpts{
+				LBClasses:      map[string]*LBClass{"lbclassKey": {MemberSubnetID: "lb-class-member-subnet-id-5678"}},
+				MemberSubnetID: "lb-class-memberSubnetId",
+			},
+		},
+	}
+
+	type fields struct {
+		lbaas LbaasV2
+	}
+	type args struct {
+		service *corev1.Service
+		svcConf *serviceConfig
+	}
+	tests := []struct {
+		name    string
+		fields  fields
+		args    args
+		want    string
+		wantErr error
+	}{
+		{
+			name:   "get member subnet id from service annotation",
+			fields: fields{},
+			args: args{
+				service: &corev1.Service{
+					ObjectMeta: v1.ObjectMeta{
+						Annotations: map[string]string{
+							ServiceAnnotationLoadBalancerMemberSubnetID: "member-subnet-id",
+						},
+					},
+				},
+				svcConf: &serviceConfig{},
+			},
+			want:    "member-subnet-id",
+			wantErr: nil,
+		},
+		{
+			name: "get member subnet id from config class",
+			fields: fields{
+				lbaas: *lbaas,
+			},
+			args: args{
+				service: &corev1.Service{
+					ObjectMeta: v1.ObjectMeta{
+						Annotations: map[string]string{
+							ServiceAnnotationLoadBalancerClass: "lbclassKey",
+						},
+					},
+				},
+				svcConf: &serviceConfig{},
+			},
+			want:    "lb-class-member-subnet-id-5678",
+			wantErr: nil,
+		},
+		{
+			name:   "get member subnet id from default config",
+			fields: fields{},
+			args: args{
+				service: &corev1.Service{},
+				svcConf: &serviceConfig{},
+			},
+			want:    "",
```

Comment:
Can you populate the config to make this have a value? It's `opts.MemberSubnetID`.

Reply:
i want to bring you attention to the import in this method, i am testing 

the `svcConfig *serviceConfig` function parameter was never used, i don't know if it is intentional but i tried finding a use case for it in the method and did not see any